caseworker/users/views/users.py

```
@@ -139,3 +141,11 @@ def post(self, request, **kwargs):
         put_gov_user(request, str(kwargs["pk"]), json={"status": request.POST["status"]})
 
         return redirect("/users/")
+
+
+class UserCaseNoteMentions(LoginRequiredMixin, TemplateView):
+    def get(self, request, **kwargs):
+        data, _ = get_user_case_note_mentions(request)
```

This doesn't seem to do any response code checking

---

you mean have it as a cached property ?

---

added as cached property

---

I don't see how `cached_property` helps.  We still aren't checking that the API response was 2XX.  If it's not we should raise an exception, could even just be something like https://github.com/uktrade/lite-frontend/blob/ab5775efd50a818bf397a1bb61c2156a9abf5298/caseworker/cases/services.py#L18

---

ahh ok miss-understood changed